docs/content/latest/releases/whats-new/latest-release.md

```diff
@@ -0,0 +1,132 @@
+* [[7225](https://github.com/yugabyte/yugabyte-db/issues/7225)] [[7228](https://github.com/yugabyte/yugabyte-db/issues/7228)] [UI] Enable the OpenShift tab, marked as beta
+
+#### Core Database
+
+* [[2717](https://github.com/yugabyte/yugabyte-db/issues/2717)] YSQL: Support ALTER FUNCTION command
+* [[6985](https://github.com/yugabyte/yugabyte-db/issues/6985)] YSQL: Add simple UNIQUE column
+
+### Improvements
+
+#### Yugabyte Platform
+
+* [[5628](https://github.com/yugabyte/yugabyte-db/issues/5628)] [Azure] Support regions with no availability zones
+* [[5807](https://github.com/yugabyte/yugabyte-db/issues/5807)] Add toggle to show deleted backups
+* [[6373](https://github.com/yugabyte/yugabyte-db/issues/6373)] Pause/Resume universe
+* [[6962](https://github.com/yugabyte/yugabyte-db/issues/6962)] Expose endpoint for downloading log files
+* [[7475](https://github.com/yugabyte/yugabyte-db/issues/7475)] Use more recent CentOS-7 base image for GCP universe VMs
+
+#### Core Database
+
+* [[7121](https://github.com/yugabyte/yugabyte-db/issues/7121)] Extend yb-admin restore_snapshot to use a custom time
+* [[7126](https://github.com/yugabyte/yugabyte-db/issues/7126)] PITR: Introduce snapshot schedule
+* [[7366](https://github.com/yugabyte/yugabyte-db/issues/7366)] YSQL: Allow getting current SQL query in pggate for debug logging
+* [[7418](https://github.com/yugabyte/yugabyte-db/issues/7418)] [[7463](https://github.com/yugabyte/yugabyte-db/issues/7463)] YSQL: Import the 'force' option for the Drop Database command
+
+### Known Issues
+
+#### Yugabyte Platform
+
+N/A
+
+#### Core Database
+
+N/A
+
+## Notes
+
+{{< note title="New release versioning" >}}
+
+Starting with v2.2, Yugabyte release versions follow a [new release versioning convention](../../versioning). The latest release series, denoted by `MAJOR.ODD`, incrementally introduces new features and changes and is intended for development and testing only. Revision releases, denoted by `MAJOR.ODD.REVISION` versioning, can include new features and changes that might break backwards compatibility. For more information, see [Supported and planned releases](../../releases-overview).
+
+{{< /note >}}
+
+## Not yet categorized
+
+* [[4944](https://github.com/yugabyte/yugabyte-db/issues/4944)] Remove TTL-related computations from the PgSQL read path
+* [[4944](https://github.com/yugabyte/yugabyte-db/issues/4944)] Remove table_tombstone_time_ from DocRowwiseIterator and delegate state to DocDBTableReader
+* [[4944](https://github.com/yugabyte/yugabyte-db/issues/4944)] docdb - Pull DocDBTableReader into DocRowwiseIterator
+* [[4944](https://github.com/yugabyte/yugabyte-db/issues/4944)] docdb - Refactor table tombstone and ancestor scanning logic from doc_reader.cc
```

Comment:
db perf related work: bunch of refactors in order to remove TTL related computations from the YSQL read path